examples/runners/ash/src/main.rs

```diff
@@ -32,6 +32,18 @@ macro_rules! offset_of {
         }
     }};
 }
+
+#[allow(dead_code)]
+struct ShaderConstants {
+    width: u32,
+    height: u32,
+    time: f32,
+}
+
+unsafe fn any_as_u8_slice<T: Sized>(p: &T) -> &[u8] {
+    ::std::slice::from_raw_parts((p as *const T) as *const u8, ::std::mem::size_of::<T>())
+}
```

Comment:
Not relevant for this PR, but we should start developing a best practices guide in the rust-gpu dev guide on how to handle stuff like this. Just letting people run around with this level of terrifying unsaftey is sure to end in horror. (stuff like `#[repr(C)]`, zero-copy serialization crates(?), etc.)